ui/app/components/app/connected-accounts-list/index.scss

```diff
@@ -0,0 +1,161 @@
+.connected-accounts-list {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__identicon {
+    margin-right: 8px;
+  }
+
+  &__account-name {
+    font-weight: bold;
+    font-size: 14px;
+    line-height: 20px;
+  }
+
+  &__primary {
+    font-style: italic;
+  }
+
+  &__account-status {
+    color: $Grey-500;
+  }
+
+  &__account-status-link {
+    &, &:hover {
+      color: $curious-blue;
+      cursor: pointer;
+    }
+  }
+
+  &__primary, &__account-status {
+    font-size: 12px;
+    line-height: 17px;
```

Comment:
Nit: The status is a bit close to the account name - closer than in the designs.

This seems to look a bit better:

```suggestion
    line-height: 17px;
    padding-top: 4px;
```

<details>
<summary>Screenshots:</summary>

Before:

![before-padding](https://user-images.githubusercontent.com/2459287/81980817-ad9d6e00-9605-11ea-9fbf-066400de4cc0.png)

After:

![after-padding](https://user-images.githubusercontent.com/2459287/81980823-af673180-9605-11ea-8459-3293f760b2bc.png)

</details>